### PR TITLE
feat(sdk): update SDK after external audit

### PR DESCRIPTION
**Pull request by 0x6e616d**

### CODE REVIEW

File: packages/tokamak/sdk/src/portals.ts [hunk 1]
```diff
@@ -213,51 +212,393 @@ export class Portals {
...
     }
-    const finalizedStatus = await this.getFinalizedWithdrawalStatus(
-      withdrawalMessageInfo.withdrawalHash
-    )
-    return finalizedStatus
-      ? MessageStatus.RELAYED
-      : MessageStatus.READY_FOR_RELAY
-  }
 
-  public async waitingDepositTransactionRelayedUsingL1Tx(
```
Comment (theo-learner) [inline packages/tokamak/sdk/src/portals.ts:250]: Could you check it? This function is used for deposit ERC20 and ETH in hardhat tasks.

**Review by theo-learner — COMMENTED** [review]
Thanks. I leaved comments in your code. And can you bump package version? We have already publish v0.0.9 to npm, but this version is v0.0.8.

File: packages/tokamak/sdk/src/portals.ts (hunk 1 above)
Comment (0x6e616d) [reply packages/tokamak/sdk/src/portals.ts:250]: Sorry, I updated

Comment (0x6e616d) [reply packages/tokamak/sdk/src/portals.ts:250]: About the version, I think after this PR, I will ping Steven to help me pump the new version of SDK.
Let him manage the version of our libraries

**Comment by theo-learner** [comment]
I have checked all tasks are working well. Thank you so much. 
@steven94kr Do you have any suggestion for versioning of sdk package?

**Review by theo-learner — APPROVED** [review]

**Comment by steven94kr** [comment]
@0x6e616d 
What I was talking about last time was about the release tag. (I remember you push tag before)

I will attach the release tag and release it.

It is good to use the npm version when the code is modified. (Sometimes it's missing, haha)
It seems better to change it to 0.0.9.

Can you do this for me? 🙇🏻‍♂️

**Comment by 0x6e616d** [comment]
> @0x6e616d What I was talking about last time was about the release tag. (I remember you push tag before)
> 
> I will attach the release tag and release it.
> 
> It is good to use the npm version when the code is modified. (Sometimes it's missing, haha) It seems better to change it to 0.0.9.
> 
> Can you do this for me? 🙇🏻‍♂️

I update two packages version:
- @tokamak-network/sdk to v0.0.9
- @tokamak-network/contract-utils to v0.0.2

**Comment by steven94kr** [comment]
Thank you very much!
I think we can merge it!

**Pull request merged**